### DISCUSSION TITLE
VaultWarden update function results in failure

### DISCUSSION THREAD

**Discussion by bobloadmire** [tteck/Proxmox#3960] [Help Needed]
I have increased ram to 5120 and cores to 4 as reccommended. monitored during the process, ram never even reached 50% utilization. results in error: "[ERROR] in line 85: exit code 0: while executing command cargo build --features "sqlite,mysql,postgresql" --release &> /dev/null"

any advice?

**Comment by darlezior** [comment]
I had same issue while updating last versione..

**Comment by MickLesk** [comment]
Can you post an correct output in verbose mode? Thats not helpful.
